docs/zh_cn/03-benchmark/benchmark.md

```diff
@@ -992,6 +992,78 @@ GPU: ncnn, TensorRT, PPLNN
     <td align="center">0.7949</td>
     <td align="center">0.7950</td>
   </tr>
+  <tr>
+    <td align="center" rowspan="3"><a href="https://github.com/open-mmlab/mmocr/tree/master/configs/textdet/psenet/psenet_r50_fpnf_600e_icdar2015.py">PSENet</a></td>
+    <td align="center" rowspan="3">TextDetection</td>
+    <td align="center" rowspan="3">ICDAR2015</td>
+    <td align="center">recall</td>
+    <td align="center">0.7526</td>
+    <td align="center">0.7526</td>
+    <td align="center">0.7526</td>
+    <td align="center">0.7526</td>
+    <td align="center">0.7520</td>
+    <td align="center">0.7496</td>
+    <td align="center">-</td>
+    <td align="center">0.7526</td>
+  </tr>
+  <tr>
+    <td align="center">precision</td>
+    <td align="center">0.8669</td>
+    <td align="center">0.8669</td>
+    <td align="center">0.8669</td>
+    <td align="center">0.8669</td>
+    <td align="center">0.8668</td>
+    <td align="center">0.8550</td>
+    <td align="center">-</td>
+    <td align="center">0.8669</td>
+  </tr>
+  <tr>
+    <td align="center">hmean</td>
+    <td align="center">0.8057</td>
+    <td align="center">0.8057</td>
+    <td align="center">0.8057</td>
+    <td align="center">0.8057</td>
+    <td align="center">0.8054</td>
+    <td align="center">0.7989</td>
+    <td align="center">-</td>
+    <td align="center">0.8057</td>
+  </tr>
+  <tr>
+    <td align="center" rowspan="3"><a href="https://github.com/open-mmlab/mmocr/tree/master/configs/textdet/panet/panet_r18_fpem_ffm_600e_icdar2015.py">PANet</a></td>
```

Comment:
may remove `align="center"` for config column and use default left alignment setting.

Reply:
`align="center"` is aligned with other settings of Chinese benchmark.